packages/app-desktop/gui/NoteListItem.tsx

```diff
@@ -149,12 +149,21 @@ function NoteListItem(props: NoteListItemProps, ref: any) {
 		color: theme.color,
 	};
 	const watchedIcon = props.isWatched ? null : <i style={watchedIconStyle} className={'fa fa-share-square'}></i>;
+	const classNames = [
+		'list-item-container',
+		props.isSelected && 'selected',
+		item.todo_completed && 'todo-completed',
+		item.is_todo ? 'todo' : 'note',
+		(props.index + 1) % 2 === 0 ? 'even' : 'odd',
```

Comment:
generic terms like "even" or "odd" on their own are fine though. I guess it's just when naming a component that we should use the "something-type" format.